packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -10,6 +10,8 @@ import { Flex } from "../Flex"
 import { Spacer } from "../Spacer"
 import { Serif } from "../Typography"
 
+const FIXED_BUTTON_HEIGHT = "92px"
```

Comment:
Why not `space(9)`?

Reply:
It's not a reference to a styled system spacing, but a specific pixel perfect measured height. Not a biggie but I would have opted for the specific height over the styled system height as the latter is used for keeping consistency, which was not the goal here. Updated using the `space` API as I'm mostly impartial.

Reply:
Makes sense; thinking here is that at scale, the less variables in the system (`space(9)` * n == 1 variable; `92px` == 1 variable) makes for more consistency visually (at scale). Since 92 is just two pixels away from our theme value, rounding down can be considered and effort towards the greater good.